code/datums/diseases/transformation.dm

```diff
@@ -97,8 +97,8 @@
 			new_mob.ghostize(can_reenter_corpse = FALSE)
 			new_mob.key = null
 
-/datum/disease/transformation/jungle_fever
 	name = "Jungle Fever"
+/datum/disease/transformation/jungle_fever
```

Comment:
Messed up proc header. Move it back up top.

Reply:
probably a misclick. Fixed.